engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -2505,6 +2520,12 @@ namespace dmGameObject
 
         if (update_context->m_FixedUpdateFrequency != 0)
         {
+            if (collection->m_FirstUpdate)
+            {
+                collection->m_FirstUpdate = 0;
+                collection->m_FixedAccumTime = update_context->m_AccumFrameTime * update_context->m_TimeScale;
+            }
```

Comment:
If it's the first update, we want to make the accumulated time the same as the parent collection.

That way, if the time scale is 1, we can get the fixed update calls in the same frame.